src/validation-schemas/login/login.ts

```diff
@@ -0,0 +1,17 @@
+import * as yup from 'yup';
```

Comment:
Let's move this file to validation-schemas/auth/login.ts and also rename this scheme to loginValidationSchema